pkgs/servers/mail/mailman/default.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, fetchurl, python, pythonPackages }:
+{ stdenv, fetchurl, python2, python2Packages }:
```

Comment:
This should better only be `python2` and instead of `python2Packages` we can then use `python2.pkgs`.